Makefile

```
@@ -0,0 +1,47 @@
+# Makefile for Agent Factory
+
+# Configuration
+DOCKER_IMAGE = agent-factory
+DOCKER_CONTAINER = agent-factory-a2a
+DOCKER_TAG = latest
+A2A_SERVER_HOST = localhost
+A2A_SERVER_PORT = 8080
```

```suggestion
A2A_SERVER_LOCAL_PORT = 8080
```

Just to distinguish it from the one the agent in the container listens to.